src/Core/Scheduler/Providers/TileProvider.js

```diff
@@ -54,7 +56,29 @@ TileProvider.prototype.executeCommand = function executeCommand(command) {
     tile.layers.set(command.threejsLayer);
 
     if (parent) {
-        parent.worldToLocal(params.center);
+        // params.center is built using a default sized ellipsoid (from Coordinates.js)
+        // It doesn't account for matrix transformation so we can't simply use:
+        //    parent.worldToLocal(params.center)
+        // to transform it in parent's local coordinates.
+        // (aka: params.center is expressed in command.layer.object3d local coords)
+
+        // So starting from:
+        //    M = parent.matrixWorld -> transform from parent local to world coord
+        //    N = command.layer.object3d.matrixWorld -> transform from local to world
+        // we compute:
+        //    N^-1 -> transform from world to command.layer.object3d local
+        // then
+        //    N^-1 * M -> parent local to command.layer.object3d local
+        // then we inverse this:
+        //    (N^-1 * M)^-1 =
+        //        M^-1 * N -> command.layer.object3d local to parent local
+
+        // tempMatrix holds M^-1
+        tempMatrix.getInverse(parent.matrixWorld);
+        // tempMatrix now holds M^-1 * N
+        tempMatrix.multiply(command.layer.object3d.matrixWorld);
```

Comment:
Normally, `layer.object3d` transformation should already be inherited via children.

I'll delete `tempMatrix.multiply(command.layer.object3d.matrixWorld)`